roles/powervs_install_services/tasks/main.yml

```diff
@@ -96,17 +96,24 @@
         state: latest
 
     # modify /etc/chronyd file
-    - name: Replace iburst line in /etc/chronyd file
+    - name: Replace iburst line in /etc/chrony.conf file
```

Comment:
Replace startum may be ?

Reply:
and this

Reply:
@sahityajain123 there are 2 tasks with same name. May this can have different name?